### PR TITLE
Complete the renaming of download-env-root.sh to debian-setup.sh.

### PR DESCRIPTION
**Pull request by fidergo-stephane-gourichon**
Commit 63e30284a257de04848869cb1cbd1ca5f6c55c9d
Date:   2019-05-03 19:39:22 -0700

>      * Rename `download-env-root.sh` to `debian-setup.sh` (should stop
>        people thinking we are Debian only).

File was renamed but old name appears in scripts (one real, one in a commented-out section).

This PR fixes.

### CODE REVIEW

File: scripts/enter-env.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ if [ -f /etc/udev/rules.d/99-hdmi2usb-permissions.rules -o -f /lib/udev/rules.d/
 	true
 else
 	echo "Please install the HDMI2USB udev rules."
-	echo "These are installed by scripts/download-env-root.sh"
+	echo "These are installed by scripts/debian-setup.sh"
```
Comment (mithro) [inline scripts/enter-env.sh]: Maybe reword this to "On debian, these can be installed by using scripts/debian-setup.sh"?

Comment (fidergo-stephane-gourichon) [reply scripts/enter-env.sh]: Done. Also offering hint about the already-existing `HDMI2USB_UDEV_IGNORE` environment variable.

I'm surprised the Travis reported failure. This can't be caused this PR changes, can it?

Comment (fidergo-stephane-gourichon) [reply scripts/enter-env.sh]: https://travis-ci.org/timvideos/litex-buildenv/builds/597670776 seems to mean that only some targets fail.

Started to fail on commit 534513f2fd6a2f5924c3d3f46237c5db946a856e which also has unrelated changes.

Is something broken on the CI scripts?

**Comment by mithro** [comment]
It looks like nextpnr is segfaulting :-/
```
terminate called after throwing an instance of 'std::out_of_range'
  what():  stoi
build_top.sh: line 4:   809 Aborted                 (core dumped) nextpnr-ice40 --hx8k --package ct256 --pcf top.pcf --json top.json --asc top.txt --pre-pack top_pre_pack.py
Traceback (most recent call last):
  File "./make.py", line 164, in <module>
    main()
  File "./make.py", line 148, in main
    vns = builder.build(**dict(args.build_option))
  File "/home/travis/build/timvideos/litex-buildenv/third_party/litex/litex/soc/integration/builder.py", line 190, in build
    toolchain_path=toolchain_path, **kwargs)
  File "/home/travis/build/timvideos/litex-buildenv/third_party/litex/litex/soc/integration/soc_core.py", line 494, in build
    return self.platform.build(self, *args, **kwargs)
  File "/home/travis/build/timvideos/litex-buildenv/third_party/litex/litex/build/lattice/platform.py", line 33, in build
    return self.toolchain.build(self, *args, **kwargs)
  File "/home/travis/build/timvideos/litex-buildenv/third_party/litex/litex/build/lattice/icestorm.py", line 184, in build
    _run_script(script)
  File "/home/travis/build/timvideos/litex-buildenv/third_party/litex/litex/build/lattice/icestorm.py", line 65, in _run_script
    raise OSError("Subprocess failed")
OSError: Subprocess failed
```
